bin/update-exercises.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# This script will update spago.dhall and package.dhall of all exercises
+# using the master files from the project template (template/).
+
+set -o pipefail
+set -u
+
+if [ $# != 1 ]; then
```

Comment:
Do we need this check or is assuming they are at `../exercises` enough?

Reply:
We could and we already do that in other places (see `scripts/ci`). I find it is a bit more effort to get right. I tend to run these scripts from various directories, i.e. the base directory or from `./bin` or some other place. The script then needs to find it's base directory and build a path to `./exercises`. You get something like:

```
base_dir=$(builtin cd "${BASH_SOURCE%/*}/.." || exit; pwd)
exercises_dir=${base_dir}/exercises
```

It works but from what I've read its not very reliable or portable. You'd think this would be trivial but it doesn't appear to be. :)

Anyway I don't have a preference either way...

Reply:
> It works but from what I've read its not very reliable or portable.

Interesting. This has worked fine for me:

```
base_dir=$(dirname $base_dir)
cd $base_dir
```

Granted, only on Linux, so yeah...

> Anyway I don't have a preference either way...

Agreed, just thought it would be better to stay consistent, since it seems fine in other scripts.

Reply:
Looks like I was already doing this in that script anyway. :) Thanks for the good feedback. I've pushed a change now.